framework/src/executioners/FixedPointSolve.C

```diff
@@ -569,12 +490,15 @@ FixedPointSolve::printFixedPointConvergenceReason()
     case MooseFixedPointConvergenceReason::CONVERGED_RELATIVE:
       _console << "CONVERGED_RELATIVE";
       break;
-    case MooseFixedPointConvergenceReason::CONVERGED_CUSTOM:
-      _console << "CONVERGED_CUSTOM";
+    case MooseFixedPointConvergenceReason::CONVERGED_PP:
+      _console << "CONVERGED_PP";
       break;
     case MooseFixedPointConvergenceReason::REACH_MAX_ITS:
       _console << "REACH_MAX_ITS";
       break;
+    case MooseFixedPointConvergenceReason::CONVERGED_OBJECT:
+      _console << "CONVERGED_OBJECT";
```

Comment:
```suggestion
      _console << "CONVERGED (see convergence object)";
```
?
I understand `CONVERGED_OBJECT` but maybe not the most user-friendly?

Reply:
I'll do `CONVERGED_OBJECT (see Convergence object)`